apps/gauzy/src/app/pages/employees/employees.component.html

```
@@ -128,7 +130,14 @@ <h4>
     }
     <ng-template [ngxPermissionsOnly]="[PermissionsEnum.ORG_EMPLOYEES_EDIT, PermissionsEnum.ORG_EMPLOYEES_VIEW]">
       <ng-template ngxPermissionsOnly="ORG_EMPLOYEES_VIEW">
-        <button status="basic" class="action secondary" size="small" nbButton underConstruction>
+        <button
+          status="basic"
```

HTMLHint (1.9.2)

[warning] The type attribute must be present on <button> elements.

(button-type-require)